Lombiq.Hosting.MediaTheme.Deployer/Program.cs

```diff
@@ -0,0 +1,172 @@
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Globalization;
+using System.IO.Compression;
+using System.Reflection;
+using static System.Console;
+
+namespace Lombiq.Hosting.MediaTheme.Deployer;
+
+internal static class Program
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage(
+        "Globalization",
+        "CA1303:Do not pass literals as localized parameters",
+        Justification = "It's a console application it doesn't need localization.")]
+    public static void Main(string[] args)
+    {
+        if (args.Length == 3 && bool.TryParse(args[2], out bool clearMediaFolder))
+        {
+            // Creating directory for the deployment.
+            var currentRootDirectory = Directory.GetDirectoryRoot(Directory.GetCurrentDirectory());
+            var newDirectoryPath = Path.Join(currentRootDirectory, "MediaThemeDeployment_")
```

Comment:
It should accept a parameter for the target directory. It can fall back to this one if it's not given.